codechain/__version__.py

```diff
@@ -0,0 +1,7 @@
+__title__ = "codechain"
+__description__ = "Python HTTP for Humans."
+__url__ = "https://codechain.io/"
+__version__ = "1.0.0"
+__author__ = "Kodebox"
+__author_email__ = "codechain@codechain.io"

```

Comment:
We use this for sdk-js: `CodeChain Team <hi@codechain.io>`

Reply:
I fixed it.